repository_service_tuf_worker/repository.py

```diff
@@ -823,22 +833,27 @@ def bump_snapshot(self) -> bool:
 
         return True
 
-    def bump_online_roles(self) -> bool:
-        """Bump online Roles (Snapshot, Timestamp, BINS)."""
-        timeout = int(self.refresh_settings().get("LOCK_TIMEOUT", 60.0))
-        logging.debug(f"Configured timeout: {timeout}")
+    def bump_online_roles(self, force: Optional[bool] = False) -> bool:
```

Comment:
As we have discussed with @kairoaraujo, I think we can:
1. move the `LOCK_TARGETS` and bootstrap check inside `bump_target_roles`
2. remove this function `bump_online_roles`
3. rename `bumpt_target_roles` to `bump_online_roles`
4. remove the `LOCK_TARGETS` lock from `publish_targets` as it will already exist in the new `bump_online_roles`.

Reply:
After another discussion with Kairo, we realized that we cannot entirely remove `bump_online_roles` and merge it into `bump_target_roles`.
We need `bump_target_roles` without a lock when we call it inside `metadata_rotations` as we add a lock there that's also used to protect the operation of persisting root.